source/Minable.cpp

```diff
@@ -121,6 +123,12 @@ void Minable::Place(double energy, double beltRadius)
 	// Calculate the object's initial position.
 	radius = scale / (1. + eccentricity * cos(theta));
 	position = radius * Point(cos(theta + rotation), sin(theta + rotation));
+
+	// Choose a random hull value for the object.
+	hull += (2. * Random::Real() - 1.) * hullDeviation;
+	// Don't allow a newly placed minable to immediately explode.
+	if(hull < 0.)
+		hull = 0.;
```

Comment:
I think that we should implement this new attribute similar to how random velocity and random lifetime are implemented. That is, the random values only ever add to the base value. This would mean we don't need to check for "hull deviation" kicking the hull into the negative, since it could only ever add to the base value (whose lowest value should be zero, but it looks like we don't check for that right now).

And in having the same behavior as those attributes, I imagine it should have the same naming scheme as well: "random hull".